lib/modules/datasource/index.ts

```diff
@@ -415,6 +416,60 @@ export async function getPkgReleases(
             version.matches(constraintValue, releaseConstraint)
         );
       });
+    } else {
+      if (constraintName === 'php') {
+        const composerVersioning = allVersioning.get('composer');
+        if (composerVersioning.isValid(constraintValue)) {
+          const lookupConfig: GetPkgReleasesConfig = {
+            datasource: GithubTagsDatasource.id,
+            depName: 'php/php-src',
+            extractVersion: '^php-(?<version>\\S+)',
+          };
+          const phpVersions = (
+            await getPkgReleases(lookupConfig)
+          )?.releases.map((release) => release.version);
+          if (!phpVersions) {
+            logger.warn('Could not fetch php releases for compatibility check');
+            return null;
+          }
```

Comment:
This is a huge problem if it ever happens, not sure whether to throw an ExternalHostError or not

Reply:
You can cache this right? Because having to look this up for every single PHP run sounds wasteful. And if it happens, blow out and try again later?